src/Widget/Settings/Publishers.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace CultuurNet\ProjectAanvraag\Widget\Settings;
+
+/**
+ * Provides a publishers type
+ */
+class Publishers
+{
+
+    /**
+     * Cleanup the configuration.
+     */
+    public function cleanup($configuration)
+    {
+        if (isset($configuration) && is_array($configuration)) {
+            return $configuration;
```

Comment:
I have a hard time understanding what this is supposed to do

Reply:
We created a new class, so we could it as a type in the annotations, since it is not possible to use an array as a type in the annotations.

See the annotations of searchResults.php
`  "publishers": "CultuurNet\ProjectAanvraag\Widget\Settings\Publishers" `